articles/api/authentication/_multifactor-authentication.md

```diff
@@ -0,0 +1,644 @@
+
+First, request a challenge based on the challenge types supported by the application and user. If you know that one-time password (OTP) is supported, you can skip the challenge request.
+
+Next, verify the multifactor authentication using the `/oauth/token` endpoint and the specified challenge type: a one-time password (OTP), a recovery code, or an out-of-band (OOB) challenge.
+
+For more information, check out:
+
+- [Multifactor Authentication and Resource Owner Password](/api-auth/tutorials/multifactor-resource-owner-password)
+- [Multifactor Authentication API](/multifactor-authentication/api)
+- [Multifactor Authentication in Auth0](/multifactor-authentication)
+
+## Challenge request
+
+```http
+POST https://${account.namespace}/mfa/challenge
+Content-Type: application/json
+{
+  "client_id": "${account.clientId}",
+  "client_secret": "YOUR_CLIENT_SECRET",
+  "mfa_token": "MFA_TOKEN",
+  "challenge_type": "oob|otp"
+}
+```
+
+```shell
+curl --request POST \
+  --url 'https://${account.namespace}/mfa/challenge' \
+  --header 'content-type: application/json' \
+  --data '{"mfa_token":"MFA_TOKEN", "challenge_type":"oob otp", "client_id": "${account.clientId}", "client_secret": "YOUR_CLIENT_SECRET"}'
+```
+
+```javascript
+var request = require("request");
+
+var options = { method: 'POST',
+  url: 'https://${account.namespace}/mfa/challenge',
+  headers: { 'content-type': 'application/json' },
+  body:
+   { mfa_token: 'MFA_TOKEN',
+     challenge_type: 'oob otp',
+     client_id: '${account.clientId}',
+     client_secret: 'YOUR_CLIENT_SECRET' },
+  json: true };
+
+request(options, function (error, response, body) {
+  if (error) throw new Error(error);
+
+  console.log(body);
+});
+```
+
+> RESPONSE SAMPLE FOR OTP:
+```JSON
+HTTP/1.1 200 OK
+Content-Type: application/json
+{
+  "challenge_type":"otp",
+}
+```
+
+> RESPONSE SAMPLE FOR OOB WITHOUT BINDING METHOD:
+```JSON
+HTTP/1.1 200 OK
+Content-Type: application/json
+{
+  "challenge_type":"oob",
+  "oob_code": "abcde...dasg"
+}
+```
+
+> RESPONSE SAMPLE FOR OOB WITH BINDING METHOD:
+```JSON
+HTTP/1.1 200 OK
+Content-Type: application/json
+{
+  "challenge_type":"oob",
+  "binding_method":"prompt",
+  "oob_code": "abcde...dasg"
+}
+```
+
+<%= include('../../_includes/_http-method', {
+  "http_badge": "badge-success",
+  "http_method": "POST",
+  "path": "/mfa/challenge",
+  "link": "#multifactor-authentication"
+}) %>
+
+This endpoint lets you request a challenge based on the challenge types supported by the application and the end user. The challenge type indicates the channel or mechanism on which to get the challenge and thus prove possession.
+
+For details on the supported challenge types refer to [Multifactor Authentication and Resource Owner Password](/api-auth/tutorials/multifactor-resource-owner-password).
```

Comment:
It's only 2 values, it would be easier if you list them here (with a short sentence explaining what it is) than redirect to another doc